documentation/pom.xml

```diff
@@ -34,102 +34,98 @@
+                  <workDirectory>target/assembly/work</workDirectory>
+                  <tarLongFileMode>posix</tarLongFileMode>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.asciidoctor</groupId>
+            <artifactId>asciidoctor-maven-plugin</artifactId>
+            <version>${asciidoctor-maven-plugin.version}</version>
+            <configuration>
+              <skip>${skipDocs}</skip>
+              <enableVerbose>true</enableVerbose>
+              <logHandler>
+                <failIf>
+                  <severity>WARN</severity>
+                </failIf>
+              </logHandler>
+              <sourceDirectory>src/main/asciidoc</sourceDirectory>
+              <preserveDirectories>true</preserveDirectories>
+              <attributes>
+                <generated-dir>${project.basedir}/../target/asciidoc/generated</generated-dir>
+                <icons>font</icons>
+                <sectanchors>true</sectanchors>
+                <!-- set the idprefix to blank -->
+                <idprefix/>
+                <idseparator>-</idseparator>
+                <docinfo1>true</docinfo1>
+                <!-- avoid content security policy violations -->
+                <skip-front-matter>true</skip-front-matter>
+                <quickstart-clone-url>${extension-url}/tree/master/quickstart</quickstart-clone-url>
+                <reference-conf-url>${datastax-driver-documentation-url}/core/configuration/reference/</reference-conf-url>
+                <fruits-html-url>${extension-url}/tree/master/quickstart/src/main/resources/META-INF/resources/fruits.html</fruits-html-url>
+                <mapper-docs-url>${datastax-driver-documentation-url}/mapper</mapper-docs-url>
+              </attributes>
+            </configuration>
+            <executions>
+              <execution>
+                <id>output-html</id>
+                <phase>prepare-package</phase>
+                <goals>
+                  <goal>process-asciidoc</goal>
+                </goals>
+                <configuration>
+                  <skip>${skipDocs}</skip>
+                  <backend>html5</backend>
+                  <attributes>
+                    <source-highlighter>coderay</source-highlighter>
+                    <!-- avoid content security policy violations -->
+                    <linkcss>true</linkcss>
+                    <copycss>true</copycss>
+                  </attributes>
+                </configuration>
+              </execution>
+            </executions>
+            <dependencies>
+              <dependency>
+                <groupId>org.asciidoctor</groupId>
+                <artifactId>asciidoctorj</artifactId>
+                <version>${asciidoctorj.version}</version>
+              </dependency>
+            </dependencies>
+          </plugin>
           <plugin>
```

Comment:
You are repeating the plugin declaration twice. You should instead merge the 2 executions into one single plugin declaration.